hat/assets/js/apps/Iaso/domains/workflows/details.tsx

```diff
@@ -94,10 +94,10 @@ export const Details: FunctionComponent<Props> = ({ router }) => {
         isLoading: boolean;
     } = useGetWorkflowVersion(versionId);
 
-    useEffect(() => {
-        if (workflowVersion?.follow_ups) {
+    const setCurrentFollowUps = workflowVersionFollowUps => {
```

Comment:
last nitpick: use a name like `updateCurrentFollowUps` so that we don't confuse your function with a `setState` function. It's the same idea as reserving the names staring with `use...` to hooks